src/components/TimeSeriesCard/TimeSeriesCard.story.jsx

```diff
@@ -2,21 +2,23 @@ import React from 'react';
 import { storiesOf } from '@storybook/react';
 import { action } from '@storybook/addon-actions';
 import { text, select, object, boolean } from '@storybook/addon-knobs';
-import memoize from 'lodash/memoize';
 
 import { COLORS, CARD_SIZES } from '../../constants/LayoutConstants';
 import { getCardMinSize } from '../../utils/componentUtilityFunctions';
-import { getIntervalChartData as getFakeData, chartData } from '../../utils/sample';
+import { getIntervalChartData, chartData } from '../../utils/sample';
 
 import TimeSeriesCard from './TimeSeriesCard';
 
-const getIntervalChartData = memoize(getFakeData);
```

Comment:
I think this still needs to be memoized so that the data isn't regenerated every time a knob changes. Currently flipping the gmt knob gets new random data every time (see the y axis changing)

![2020-06-09 16-17-21](https://user-images.githubusercontent.com/3360588/84200806-e8f85480-aa6c-11ea-9121-d74dae9f9f51.gif)

Reply:
unfortunately @tay1orjones if I leave the memoize in, then when you switch between chart stories, the data never changes.  I figured the knob behavior was less bad then the story switching behavior

Reply:
Yeah, static slices of functionality/data for each story is the primary intent of storybook though. 

If we want to show different data for different stories, we should probably regen the data at the individual story level instead of globally for the entire chapter/section.